integration-test/build.gradle.kts

```diff
@@ -191,7 +196,6 @@ tasks.clean {
 
 tasks.register<JavaExec>("TrinoTest") {
   classpath = sourceSets["test"].runtimeClasspath
-  systemProperty("gravitino.log.path", buildDir.path + "/integration-test.log")
```

Comment:
Why remove this line?

Reply:
Because it duplicated, It's already exist in https://github.com/datastrato/gravitino/blob/main/build.gradle.kts#L163